src/main/java/com/linkedin/xinfra/monitor/apps/SingleClusterMonitor.java

```diff
@@ -55,53 +57,81 @@
   private final TopicManagementService _topicManagementService;
   private final String _name;
   private final List<Service> _allServices;
+  private final boolean _isTopicManagementServiceEnabled;
 
   public SingleClusterMonitor(Map<String, Object> props, String name) throws Exception {
     ConsumerFactory consumerFactory = new ConsumerFactoryImpl(props);
     _name = name;
-    _topicManagementService = new TopicManagementService(props, name);
-    CompletableFuture<Void> topicPartitionResult = _topicManagementService.topicPartitionResult();
+    LOG.info("SingleClusterMonitor properties: {}", prettyPrint(props));
+    _isTopicManagementServiceEnabled = (boolean) props.get("topic-management.topicManagementEnabled");
```

Comment:
The string used to get the property should be a constant somewhere?  Isn't there some kind of Config class that can be used?